components/giscus-comments.tsx

```diff
@@ -0,0 +1,23 @@
+import Giscus from "@giscus/react";
+
+export default function Comment() {
+  return (
+  <>
+    <br />
+    <hr />
+    <br />
+    {process.env.NEXT_PUBLIC_REPO && process.env.NEXT_PUBLIC_REPO_ID && process.env.NEXT_PUBLIC_CATEGORY_ID ? (
+    <Giscus
+      repo= {process.env.NEXT_PUBLIC_REPO as `${string}/${string}`}
+      repoId= {process.env.NEXT_PUBLIC_REPO_ID}
+      category="Announcements"
+      categoryId= {process.env.NEXT_PUBLIC_CATEGORY_ID}
+      mapping="pathname"
```

Comment:
Is this what determines which discussion the comments are placed in?

Reply:
Yes, it creates a thread in the category specified based off of the path like 'blog/hello-world' for example. It might be a good idea to host and pull the comments on a separate repo.

Reply:
Ah cool, let's experiment with that in our preview environment in https://github.com/WATonomous/infra-config